syntax_and_semantics/if_var.md

```diff
@@ -35,7 +35,7 @@ Of course, reassigning a variable inside the `then` branch makes that variable h
 
 ## Limitations
 
-The above logic **doesn’t** work with instance variables, class variables and variables bound in a closure. The value of these kinds of variables could potentially be affected by another fiber after the condition was checked, rendering it `nil`.
+The above logic works **only for local variables**. It doesn’t work with instance variables, class variables, or variables bound in a closure. The value of these kinds of variables could potentially be affected by another fiber after the condition was checked, rendering it `nil`. It also does not work with constants.
```

Comment:
Why not just add constants to the existing list?

Reply:
As briefly discussed above, the reason not to do that is that the next sentence goes on to say that those kinds of variables can be affected by another fiber, a rationale which isn't valid for constants.

Reply:
my bad